test/test-helper.js

```diff
@@ -297,6 +297,12 @@ const helper = {
         );
     },
 
+    assertErrorWithName: function (err, name) {
+        assert.ok(err, "Expected to receive an error");
```

Comment:
:thinking: Shouldn't this message say more about what we received instead of an error?